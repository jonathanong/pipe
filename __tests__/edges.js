
const assert = require('assert')

const { streamToString } = require('../utils')
const render = require('../lib')

test('values at the edges', async () => {
  const result = await streamToString(render`${'1'}test${'2'}`)
  assert.strictEqual(result, '1test2')
  expect(result).toMatchSnapshot()
})
