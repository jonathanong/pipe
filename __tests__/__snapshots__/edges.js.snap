// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`values at the edges 1`] = `"1test2"`;
